database/mssql/scripts/dbscripts/PSP_PIMS_S10_00/Build/82_DML_LEASE_TESTING.sql

```diff
@@ -0,0 +1,55 @@
+INSERT INTO PIMS_ORGANIZATION (ORGANIZATION_IDENTIFIER, ORGANIZATION_NAME, ORGANIZATION_TYPE_CODE, ORG_IDENTIFIER_TYPE_CODE, ADDRESS_ID, APP_CREATE_TIMESTAMP, APP_CREATE_USERID, APP_CREATE_USER_DIRECTORY, APP_LAST_UPDATE_TIMESTAMP, APP_LAST_UPDATE_USERID, APP_LAST_UPDATE_USER_DIRECTORY)
+VALUES
+  (N'LILMOUSE', N'French Mouse Property Management',         N'REALTOR', N'OTHINCORPNO', (SELECT ADDRESS_ID FROM PIMS_ADDRESS WHERE STREET_ADDRESS_1 LIKE '%Souris%'),    CURRENT_TIMESTAMP, N'Lease Seed Data', N'Leasse Seed Data', CURRENT_TIMESTAMP, N'Leasse Seed Data', N'Leasse Seed Data'),
+  (N'DQ4EVA',   N'Dairy Queen Forever! Property Management', N'REALTOR', N'OTHINCORPNO', (SELECT ADDRESS_ID FROM PIMS_ADDRESS WHERE STREET_ADDRESS_1 LIKE '123 Main%'),   CURRENT_TIMESTAMP, N'Lease Seed Data', N'Leasse Seed Data', CURRENT_TIMESTAMP, N'Leasse Seed Data', N'Leasse Seed Data'),
+  (N'DAKATZ',   N'Pussycat Property Management',             N'REALTOR', N'OTHINCORPNO', (SELECT ADDRESS_ID FROM PIMS_ADDRESS WHERE STREET_ADDRESS_1 LIKE 'PO Box%'),     CURRENT_TIMESTAMP, N'Lease Seed Data', N'Leasse Seed Data', CURRENT_TIMESTAMP, N'Leasse Seed Data', N'Leasse Seed Data'),
+  (N'BOB123',   N'Bob''s Property Management',               N'REALTOR', N'OTHINCORPNO', (SELECT ADDRESS_ID FROM PIMS_ADDRESS WHERE STREET_ADDRESS_1 LIKE '%Hollywood%'), CURRENT_TIMESTAMP, N'Lease Seed Data', N'Leasse Seed Data', CURRENT_TIMESTAMP, N'Leasse Seed Data', N'Leasse Seed Data');
+
+-- Insert people
+INSERT INTO PIMS_PERSON (ADDRESS_ID, SURNAME, FIRST_NAME, MIDDLE_NAMES, APP_CREATE_TIMESTAMP, APP_CREATE_USERID, APP_CREATE_USER_DIRECTORY, APP_LAST_UPDATE_TIMESTAMP, APP_LAST_UPDATE_USERID, APP_LAST_UPDATE_USER_DIRECTORY)
+VALUES
+  ((SELECT ADDRESS_ID FROM PIMS_ADDRESS WHERE STREET_ADDRESS_1 LIKE '%Hollywood%'), N'Smith',  N'Bob',    N'Billy',        CURRENT_TIMESTAMP, N'Lease Seed Data', N'Leasse Seed Data', CURRENT_TIMESTAMP, N'Leasse Seed Data', N'Leasse Seed Data'),
+  ((SELECT ADDRESS_ID FROM PIMS_ADDRESS WHERE STREET_ADDRESS_1 LIKE 'PO Box%'),     N'Wilson', N'Volley', N'Ball',         CURRENT_TIMESTAMP, N'Lease Seed Data', N'Leasse Seed Data', CURRENT_TIMESTAMP, N'Leasse Seed Data', N'Leasse Seed Data'),
+  ((SELECT ADDRESS_ID FROM PIMS_ADDRESS WHERE STREET_ADDRESS_1 LIKE '123 Main%'),   N'Cheese', N'Stinky', N'',             CURRENT_TIMESTAMP, N'Lease Seed Data', N'Leasse Seed Data', CURRENT_TIMESTAMP, N'Leasse Seed Data', N'Leasse Seed Data'),
+  ((SELECT ADDRESS_ID FROM PIMS_ADDRESS WHERE STREET_ADDRESS_1 LIKE '%Souris%'),    N'Mouse',  N'Minnie', N'Nacho Cheese', CURRENT_TIMESTAMP, N'Lease Seed Data', N'Leasse Seed Data', CURRENT_TIMESTAMP, N'Leasse Seed Data', N'Leasse Seed Data');
+
+-- Insert contact information for people
+INSERT INTO PIMS_CONTACT_METHOD (PERSON_ID, CONTACT_METHOD_TYPE_CODE, CONTACT_METHOD_VALUE, APP_CREATE_TIMESTAMP, APP_CREATE_USERID, APP_CREATE_USER_DIRECTORY, APP_LAST_UPDATE_TIMESTAMP, APP_LAST_UPDATE_USERID, APP_LAST_UPDATE_USER_DIRECTORY)
```

Comment:
PERSMOBIL isn't a valid option, see below comment.

Reply:
Got ahead of the build.  I'll amend the script and the data to the correct value for your build.